app/helpers/save_this_page_helper.rb

```diff
@@ -15,6 +15,10 @@ def heading_text(options)
     options[:page_is_saved] ? I18n.t("components.save_this_page.page_was_saved_heading") : I18n.t("components.save_this_page.page_not_saved_heading")
   end
 
+  def saved_pages_link
+    "#{Plek.new.website_root}/account/saved-pages"
+  end
```

Comment:
This method isn't used anywhere any more, right?

Reply:
eugh you're right, thought I'd removed it 🤦🏻‍♀️